### PR TITLE
runs default sort column impl

### LINKED ISSUE #211: Runs default sort
The Runs table should be sorted by default by the started column descending.

Screenshot 2020-06-22 at 14 56 05

### PR DESCRIPTION
**Pull request by senelesithole**
fix #211

### CODE REVIEW

File: ui/src/app/components/runs/runs.component.html [hunk 1]
```diff
@@ -57,7 +57,7 @@
         </app-number-range-filter>
       </clr-dg-filter>
     </clr-dg-column>
-    <clr-dg-column [clrDgField]="dagRunColumns.STARTED">
+    <clr-dg-column [clrDgField]="dagRunColumns.STARTED" [clrDgField]="'name'" [clrDgSortOrder]="descSort">
```
Comment (jozefbakus) [inline ui/src/app/components/runs/runs.component.html]: ` [clrDgField]="'name'" ` is there a reason why you have it there ? 
It has been already defined `[clrDgField]="dagRunColumns.STARTED"`

**Review by kevinwallimann — COMMENTED** [review]
When I click on "Clear sort" (without sorting on another column, activating filters etc.), there is the following error message:

ERROR TypeError: Cannot set property 'sortOrder' of undefined
    at RunsComponent.clearSort (runs.component.ts:106)
    at RunsComponent_Template_button_click_7_listener (runs.component.html:25)
    at executeListenerWithErrorHandling (core.js:21683)
    at wrapListenerIn_markDirtyAndPreventDefault (core.js:21725)
    at HTMLButtonElement.<anonymous> (platform-browser.js:934)
    at ZoneDelegate.invokeTask (zone-evergreen.js:400)
    at Object.onInvokeTask (core.js:40888)
    at ZoneDelegate.invokeTask (zone-evergreen.js:399)
    at Zone.runTask (zone-evergreen.js:168)
    at ZoneTask.invokeTask [as invoke] (zone-evergreen.js:481)

File: ui/src/app/components/runs/runs.component.html (hunk 1 above)
Comment (senelesithole) [reply ui/src/app/components/runs/runs.component.html]: no reason at all. fixed

**Comment by jozefbakus** [comment]
> When I click on "Clear sort" (without sorting on another column, activating filters etc.), there is the following error message:
> 
> ERROR TypeError: Cannot set property 'sortOrder' of undefined
> at RunsComponent.clearSort (runs.component.ts:106)
> at RunsComponent_Template_button_click_7_listener (runs.component.html:25)
> at executeListenerWithErrorHandling (core.js:21683)
> at wrapListenerIn_markDirtyAndPreventDefault (core.js:21725)
> at HTMLButtonElement. (platform-browser.js:934)
> at ZoneDelegate.invokeTask (zone-evergreen.js:400)
> at Object.onInvokeTask (core.js:40888)
> at ZoneDelegate.invokeTask (zone-evergreen.js:399)
> at Zone.runTask (zone-evergreen.js:168)
> at ZoneTask.invokeTask [as invoke] (zone-evergreen.js:481)

Don't worry about this. It has been solved by fix for my comment

**Review by jozefbakus — APPROVED** [review]

**Pull request merged**